### ISSUE TITLE
Odháčkovač v názvu stránky

### ISSUE THREAD

**Issue by Aktron** [pirati-web/pirati.cz#335]
Klub se rozhodl, že založí stránky pro jednotlivé týmy. Po týmu Spravedlnost vzniká i "Místní rozvoj". Bohužel se v názvu stránky:

https://www.pirati.cz/pripoj-se/mistni-rozvoj/

zobrazuje (v tabu apod). odháčkovaná podoba "Mistni Rozvoj", která tam být nemá, vypadá to dost hrozně.

**Comment by madbeyk** [comment]
S těma titlama stránek si při tvorbě evidentně nikdo moc hlavu nedělal, na spoustě stránek to dědí default "Pirátská strana". Přitom minimálně z pohledu SEO by to chtělo systémově vyřešit.

**Comment by petrpulc** [comment]
Problém je zřejmě v tom, že generovaný `page.ogtitle` je bez háčků a čárek a `_includes/head.html` jako primární bere ogtitle i když v `md` souborech je všude definovaný `name`... Pokusil jsem se o opravu v #396

**Comment by petrpulc** [comment]
BTW, asi by nebylo špatné se nad tím opravdu trochu zamyslet, oddělit `og:title` a spíše v markdownu přepisovat titulek pomocí `title` a `name` používat pro `<h1>` a pro titlulek jako fallback. Ale tak hluboce do toho zase nevidím abych si dovolil takovou úpravu.

**Comment by petrpulc** [comment]
Jo, tak ono to bohužel není tak, že by se název zbavoval háčků ale on se titulek zjevně generuje z cesty pokud není definován `page.title` přímo. Což není skoro nikde. Ale je možné využít `fullname` a v případě jeho přítomnosti použít `name` pro titulek. Otestuji ještě pořádně; za aktuální stav se omlouvám.
